ubuntuStation/templates/base.html

```diff
@@ -34,6 +34,14 @@
 
 						</ul>
 					</li>
+					<li class="dropdown">
+						<a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Users <span class="caret"></span></a>
+						<ul class="dropdown-menu" role="menu">
+							<li><a href="{{ url_for('show_users') }}">Manage Users</a></li>
```

Comment:
The "manage users" option added to the menu should be mentioned in the original issue. I was expecting only the login page to have been changed. Others may not be aware for the same reason.